graviti/exception.py

```diff
@@ -129,18 +129,53 @@ def __str__(self) -> str:
         return super().__str__()
 
 
+RESPONSE_ERROR_DISTRIBUTOR: Dict[str, Type[ResponseError]] = {}
+
+
+_R = TypeVar("_R", bound=ResponseError)
+
+
+class ResponseErrorRegister:
+    """A class decorator to register the ResponseError into the distributor.
+
+    Arguments:
+        error_code: The response error code of the specific ResponseError.
+
+    """
+
+    def __init__(self, error_code: str) -> None:
+        self._error_code = error_code
+
+    def __call__(self, response_error: Type[_R]) -> Type[_R]:
+        """Register the ResponseError into the distributor.
+
+        Arguments:
+            response_error: The response error needs to be registered.
+
+        Returns:
+            The input response error class unchanged.
+
+        """
+        RESPONSE_ERROR_DISTRIBUTOR[self._error_code] = response_error
+
+        return response_error
+
+
+@ResponseErrorRegister("AccessDenied")
```

Comment:
Shall we put `STATUS_CODE` into the decorator?

Reply:
We do not use `STATUS_CODE` to identify the errors currently, so I don't think it is necessary to put it into the decorator.